tests/cases/tmmock2tmmock/scripts/test-tx

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -eux
+
+SCRIPT_DIR=$(cd $(dirname $0); pwd)
+RLY_BINARY=${SCRIPT_DIR}/../../../../build/yrly
+RLY="${RLY_BINARY} --debug"
+
+TM_ADDRESS0=$(${RLY} tendermint keys show ibc0 testkey)
+TM_ADDRESS1=$(${RLY} tendermint keys show ibc1 testkey)
+
+RETRY_COUNT=20
```

Comment:
Could you also change this from `20` to `5`?

Reply:
https://github.com/hyperledger-labs/yui-relayer/pull/96/commits/498008702023ac2eef6de6a83f96c5f511e047d3
Fixed